app/services/open_library_service.rb

```diff
@@ -0,0 +1,43 @@
+class OpenLibraryService
+  include HTTParty
+  base_uri 'https://openlibrary.org'
+
+  def initialize(isbn)
+    @isbn = isbn
+    @options = { query: { bibkeys: "ISBN:#{isbn}", format: 'json', jscmd: 'data' } }
+  end
+
+  def book
+    response = self.class.get('/api/books', @options)
+    format(@isbn, response)
+  end
+
+  private
+
+  def format(isbn, response)
+    response = JSON.parse(response.body)["ISBN:#{isbn}"]
+    if response.nil?
+      not_found(isbn)
+    else
+      found_record(isbn, response)
+    end
+  end
+
+  def not_found(isbn)
```

Comment:
It's a good id, but I am not thinking is very DRY. Why not return right format in case of success and in case of fail, let the controller handle the error. 
In other word: controller should return the `not_found` using the method you define in you `ApiController`